packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -244,10 +247,16 @@ class FlutterSkiaGoldFileComparator extends FlutterGoldenFileComparator {
   static bool isAvailableForEnvironment(Platform platform) {
     final String cirrusPR = platform.environment['CIRRUS_PR'] ?? '';
     final String cirrusBranch = platform.environment['CIRRUS_BRANCH'] ?? '';
-    return platform.environment.containsKey('CIRRUS_CI')
+    final bool cirrusPostSubmit = platform.environment.containsKey('CIRRUS_CI')
       && cirrusPR.isEmpty
       && cirrusBranch == 'master'
       && platform.environment.containsKey('GOLD_SERVICE_ACCOUNT');
+
+    final bool luciPostSubmit = platform.environment.containsKey('SWARMING_TASK_ID')
+      // TODO(PIINKS): I don't think this key exists in post-submit luci checks, verify.
```

Comment:
Still need to confirm this.

Reply:
Looks like this is not a good indicator for post- v pre- submit checks. @dnfield do you know if there is a way to know when these checks are executing on Luci?

Reply:
Going to try `git_url`